pytorch/buildspec.yml

```diff
@@ -152,10 +152,10 @@ images:
     tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
     docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.sagemaker., *DEVICE_TYPE ]
     context:
-      <<: *INFERENCE_CONTEXT
-  BuildSageMakerGPUPTInferencePy3DockerImage:
-    <<: *INFERENCE_REPOSITORY
-    build: &PYTORCH_GPU_INFERENCE_PY3 false
+      <<: *TRAINING_CONTEXT
```

Comment:
This should be inference context, right? The image below is also referencing the "train" name for some reason. Can you take a look?